anime_downloader/watch.py

```diff
@@ -43,6 +43,16 @@ def list(self):
                                       *anime.progress(),
                                       meta=meta.get('Type', '')))
 
+    def anime_list(self):
+        # Stores list of anime names in watcher's list
+        anime_names = []
+        animes = self._read_from_watch_file()
+
+        for idx, anime_name in enumerate(animes):
+            anime_names.append(anime_name)
+        
+        return anime_names
```

Comment:
Won't `return self._read_from_watch_file()` be enough?

Reply:
You're right. Fixed.